### ISSUE TITLE
V-click-away fires click sometimes

### ISSUE THREAD

**Issue by lucassimines** [VinceG/vue-click-away#16]
I made a modal that has a `<button>` tag inside, and just sometimes when I click on the button, the v-click-away fires the click.

**Comment by VinceG** [comment]
@fmcdigital Yeah, that seems to happen. i think we need to ignore certain elements or someone prevent it from being triggered on certain elements

**Comment by stuartcusackie** [comment]
Try adding this to your element as a temporary workaround.:
`v-on:click.stop`

I'm not sure it will work for every case

**Comment by twentytwo777** [comment]
> Try adding this to your element as a temporary workaround.: `v-on:click.stop`
> 
> I'm not sure it will work for every case

thx
